src/test/java/uk/co/ramp/covid/simulation/place/HospitalTest.java

```diff
@@ -238,4 +238,46 @@ public void testSendHome() {
         assertEquals("Unexpected people left in hospital", expPeople, hospital.getNumPeople());
     }
 
+    @Test
+    public void childrenAreAcompaniedByAdultsForApts() {
```

Comment:
Typo (or two) - Should be `childrenAreAccompaniedByAdultsForAppts()` (with double 'c' and double 'p')?